include/bluetooth/direction.h

```diff
@@ -9,20 +9,35 @@
 
 /** Constant Tone Extension (CTE) types. */
 enum bt_df_cte_type {
+	/** Angle of Arrival mode. Antenna switching done on receiver site. */
+	BT_DF_CTE_TYPE_AOA = 0,
+	/** Angle of Departure mode with 1 us antenna switching slots.
+	 *  Antenna switching done on transmitter site.
+	 */
+	BT_DF_CTE_TYPE_AOD_1US = 1,
+	/** Angle of Departure mode with 2 us antenna switching slots.
+	 *  Antenna switching done on transmitter site.
+	 */
+	BT_DF_CTE_TYPE_AOD_2US = 2,
+};
+
+/** Constant Tone Extension (CTE) types for use as bit field values */
+enum bt_df_cte_type_bits {
```

Comment:
Hmm, not sure I am a fan of having both `bt_df_cte_type` and `bt_df_cte_type_bits`, if they define the same values as a bitfield and just as values. 

Do you think it would be possible to keep only of them, and modify the API to somehow only use one of them? Similar to how we use the `BT_GAP_LE_PHY` from `gap.h`.

Reply:
Well, I should have stated my concerns before posting changes. In general I agree. 
I think that public API can use single `enum` just differentiate the usage. E.g. for `cte_type` use it as single value member; whereas for `cte_types` use it as bitfiled.

Are we have a common opinion about it?

Reply:
> Do we have a common opinion about it?

I think so, not sure I follow your comment 100% to be honest :) But it sounds like what you are proposing is the same solution we have for the `BT_GAP_LE_PHY` values

Reply:
Changes done. I've aligned other code to use the same approach.